src/strife/r_data.c

```diff
@@ -732,6 +732,8 @@ void R_InitColormaps (void)
 void R_InitData (void)
 {
     R_InitTextures ();
+    // [crispy] Initialize and generate gamma-correction levels.
+    I_SetGammaTable();
```

Comment:
Have to make small clarification here. `I_SetGammaTable()` is called slightly earlier than in other games, as `D_IntroTick()` calls right below seems to be using gfx drawing functions. Such earlier call should be safe, but forgive me, I don't want to deep into Strife code deeper, it's simply _not my_ game...